docs/reference-guides/data/data-core.md

```diff
@@ -234,6 +431,41 @@ Returns the Entity's record object by key. Returns `null` if the value is not
 yet received, undefined if the value entity is known to not exist, or the
 entity object if it exists and is received.
 
+_Usage_
+
+```jsx
+import { store as coreDataStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import { sprintf, __ } from '@wordpress/i18n';
+
+const ExampleComponent = () => {
+    const { postData, termData } = useSelect( ( select ) => {
+        return {
+            postData: select( coreDataStore ).getEntityRecord(
```

Comment:
We should consider marking `getEntityRecord` and `getEntityRecords` selectors as internal and promote related React hooks `useEntityRecord` and `useEntityRecords` instead. Those selectors still could be useful outside of React components, though. @adamziel, what do you think?

Maybe there is a way to reference those two concepts in some nice way.

Reply:
These selectors are used across many `@wordpress` packages, unfortunately React hooks won't work in custom store actions.

Reply:
>  promote related React hooks useEntityRecord and useEntityRecords instead.

This is a great idea! They're more succinct and easier to think of than `useSelect` + `getEntityRecord`